README.md

```diff
@@ -9,8 +9,10 @@ Idea from Nick Carter, initial implementation from Richard Smith.
     Converts one (or several) .ninja_log files into chrome's about:tracing format
 
     Usage:
-        rm $BUILDDIR/.ninja_log && ninja -C $BUILDDIR
+        ninja -C $BUILDDIR
         ninjatracing $BUILDDIR/.ninja_log > trace.json
 
-    (If you don't have time for a clean build, at least run
-    `ninja -C $BUILDDIR -t recompact` first.)
+    By default this will show build timing results for the most recent (possibly
+    incremental) build. To show build timing results for every target, whether
+    built in the last build or previously, use --showall. Note that this will
+    overlap multiple builds and will thus exaggerate build parallelism.
```

Comment:
Can you keep the bits that you deleted down here? Something like "If you want to measure a clean full build: $as_before"